contracts/dapis/DapiServer.sol

```diff
@@ -309,14 +309,16 @@ contract DapiServer is
     ) external view override returns (bool) {
         bytes32 beaconId = subscriptionIdToBeaconId[subscriptionId];
         require(beaconId != bytes32(0), "Subscription not registered");
-        DataFeed storage beacon = dataFeeds[beaconId];
+        // Assuming that the update value will be signed after this condition
+        // returns true, the update timestamp will be larger than
+        // `block.timestamp`, which will still satisfy the update condition.
```

Comment:
Not sure if this comment adds much value. It seems that we are just saying that the check uses block.timestamp because psp update will happen after this condition check so it will obviously satisfy heartbeat value, right?

Reply:
Yes. I try to state such assumptions explicitly because otherwise the auditor will point it out and claiming that it's intended behavior afterwards doesn't remove the issue from the audit report.